### PR TITLE
Create Docs: Guides “gridsome”

### PR DESCRIPTION
**Pull request by milos5593**

### CODE REVIEW

File: website/content/docs/gridsome.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      use: `gridsome-plugin-netlify-cms`,
+      options: {
+        publicPath: `/admin`
+      }
+    },
+  ]
+}
+```
+
+Please read [gridsome-plugin-netlify-cms](https://gridsome.org/plugins/gridsome-plugin-netlify-cms), [transformer-remark](https://gridsome.org/plugins/@gridsome/transformer-remark) for more information about the configurations.
```
Comment (erezrokah) [inline website/content/docs/gridsome.md:83]:
```suggestion
Please read [gridsome-plugin-netlify-cms](https://gridsome.org/plugins/gridsome-plugin-netlify-cms), [transformer-remark](https://gridsome.org/plugins/@gridsome/transformer-remark) for more information.
```

File: website/content/docs/gridsome.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Netlify CMS</title>
+</head>
+<body>
+  <script src="index.js" type="module"></script>
+</body>
+</html>
```
Comment (erezrokah) [inline website/content/docs/gridsome.md:104]:
```suggestion
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Netlify CMS</title>
  </head>
  <body>
    <script src="index.js" type="module"></script>
  </body>
</html>

```

File: website/content/docs/gridsome.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+```js
+import CMS from "netlify-cms"
+```
+
+Your `config.yml` for GitHub should look like this:
+
+```yml
+backend:
+  name: github
```
Comment (erezrokah) [inline website/content/docs/gridsome.md:117]: Can we use `git-gateway` like the other guides for consistency?

File: website/content/docs/gridsome.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    create: true
+    slug: "{{slug}}"
+    fields:
+      - {label: "Title", name: "title", widget: "string"}
+      - {label: "Excerpt", name: "excerpt", widget: "string"}
+      - {label: "Publish Date", name: "date", widget: "datetime"}
+      - {label: "Body", name: "body", widget: "markdown"}
+```
+
+## Add your repo to Netlify
```
Comment (erezrokah) [inline website/content/docs/gridsome.md:136]: Should we add the missing `Push to GitHub` part like the other guides?
https://www.netlifycms.org/docs/gatsby/#push-to-github

**Review by erezrokah — CHANGES_REQUESTED** [review]
@milos5593 Thank you for creating this guide. Sorry for my late response.
Added a few comments - most of them are minor.
The main issue is to have this guide aligned with other guides with regards to pushing content to GitHub and connecting to Netlify (git-gateway and Identity).

**Comment by milos5593** [comment]
@erezrokah mo problems, I'll make the changes in a couple of day's, need to finalize a project for a client.

**Comment by erezrokah** [comment]
> I'll make the changes in a couple of day's, need to finalize a project for a client.

Sure no rush :)

**Comment by milos5593** [comment]
@erezrokah  Let me know if this works.

**Review by erezrokah — APPROVED** [review]
Thanks @milos5593, I added another commit to fix some syntax highlighting issues and replaced GitHub authentication with `Identity` and `git-gateway`  as the `config.yml` uses `backend: git-gateway`.
The latter also aligns this guide with the other ones in regards with authentication method.

**Pull request merged**